wordle-web/pages/wordEditor.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+  <v-progress-linear v-if="isLoading" color="primary" indeterminate />
+  <v-card class="ma-10">
+    <v-row align="center">
+      <v-col cols="6" align="center">
+        <v-tooltip
+          text="You must be Master of the Universe and 21 or older to add words"
+          :disabled="isOlderThanTwentyOne && isMotU">
+          <template v-slot:activator="{ props }">
+            <v-btn
+              v-bind="props"
+              :variant="isOlderThanTwentyOne && isMotU ? 'elevated' : 'plain'"
+              :color="isOlderThanTwentyOne && isMotU ? 'primary' : ''"
+              class="ma-0"
+              @click="if (isOlderThanTwentyOne && isMotU) addWordDialog = true;"
+              >Add word</v-btn
+            >
+          </template>
+        </v-tooltip>
+      </v-col>
+      <v-col>
+        <v-radio-group v-model="commonRadio">
+          <v-radio
+            label="Both common/uncommon"
+            :value="CommonRadio.Both"></v-radio>
+          <v-radio label="Only common" :value="CommonRadio.Common"></v-radio>
+          <v-radio
+            label="Only uncommon"
+            :value="CommonRadio.Uncommon"></v-radio>
+        </v-radio-group>

```

Comment:
```suggestion
        <v-radio-group v-model="commonRadio">
          <v-radio
            label="Both common/uncommon"
            :value="CommonRadio.Both"/>
          <v-radio label="Only common" :value="CommonRadio.Common"/>
          <v-radio
            label="Only uncommon"
            :value="CommonRadio.Uncommon"/>
        </v-radio-group>
```
Can use self closing tags. I would say that having a radio button for showing both common and uncommon is kind of strange. These are filters so having a filter for all doesn't seem good. I would suggest making a way to remove both clicks so that if neither are clicked, all words are shown, toggle just the uncommon words. Provide a way to clear the filters

Reply:
That makes sense to me. Thank you for the suggestion!